### PR TITLE
ER-1179 Fixes trading name in readonly view

### PR DESCRIPTION
**Pull request by dev4valtech**

### CODE REVIEW

File: src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerNameServiceTest.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+            var result = sut.GetEmployerNameAsync(Guid.NewGuid()).Result;
+
+            result.Should().Be(legalEntityName);
+        }
+
+        [Theory]
+        [InlineData(VacancyStatus.Draft)]
+        [InlineData(VacancyStatus.Referred)]
+        public void ShouldReturnTradingName(VacancyStatus status)
```
Comment (sachpatel) [inline src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerNameServiceTest.cs]: async Task?

File: src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerNameServiceTest.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+            var result = sut.GetEmployerNameAsync(Guid.NewGuid()).Result;
+
+            result.Should().Be(employerName);
+        }
+
+        [Theory]
+        [InlineData(VacancyStatus.Draft)]
+        [InlineData(VacancyStatus.Referred)]
+        public void ShouldReturnVacancyLegalEntityName(VacancyStatus status)
```
Comment (sachpatel) [inline src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerNameServiceTest.cs]: async Task?

File: src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerNameServiceTest.cs (hunk 1 above)
Comment (dev4valtech) [reply src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerNameServiceTest.cs]: thought about it and went ahead with no tasks deliberately as there is no need for it.

File: src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerNameServiceTest.cs (hunk 2 above)
Comment (dev4valtech) [reply src/Shared/UnitTests/Vacancies.Client/Application/Services/EmployerNameServiceTest.cs]: thought about it and went ahead with no tasks deliberately as there is no need for it.

**Pull request merged**